app/models/course.rb

```diff
@@ -285,6 +290,10 @@ def category_article_ids
     categories.inject([]) { |ids, cat| ids + cat.article_ids }
   end
 
+  def update_in_future?
```

Comment:
Instead of this, I think it would be more useful to define `updates_until` or something like that, which is the date that updates will continue until. This will be much more useful on the frontend, so people know exactly what to expect in terms of continued updates.

Then on the frontend, you can change the message depending on whether `updates_until` is in the future or not, and you can also separately show a message when a non-updating course has been scheduled for a single update (ie, `needs_update: true`).

Reply:
Done!